var/spack/repos/builtin/packages/ncl/package.py

```diff
@@ -104,6 +107,7 @@ class Ncl(Package):
     depends_on("hdf", when="+hdf4")
     depends_on("gdal@:2.4", when="+gdal")
     depends_on("udunits", when="+udunits2")
+    depends_on("jasper@2.0.32", when="+grib")
```

Comment:
Do we need to pin versions this strictly?

Reply:
Yes, unfortunately. It's has some arcane software built in that requires that we thread the needle when it comes to jasper versions.